Documentation/Development.md

```diff
@@ -48,6 +48,23 @@ After building, the server will be located at `.build/debug/sourcekit-lsp`, or a
 
 SourceKit-LSP is designed to build against the latest SwiftPM, so if you run into any issue make sure you have the most up-to-date dependencies by running `swift package update`.
 
+### Docker
+
+To run SourceKit-LSP inside the [Swift Docker container](https://swift.org/download/#docker), you also need to install the following packages before following the instructions from the Linux section.
+
+```sh
+$ apt update
+$ apt install libsqlite3-dev libncurses5-dev python ninja-build
```

Comment:
Yes, that’s correct. I figured it’s easier o just list them all as dependencies. `python` and `ninja-build` don’t have a huge overhead so whoever is looking into this might just install them to save themselves some trouble later.